lib/fs.js

```diff
@@ -591,6 +604,36 @@ function writeSync(fd, buffer, offset, length, position) {
   return result;
 }
 
+// usage:
+// fs.writev(fd, buffers, [position], callback);
+function writev(fd, buffer, position, callback) {
```

Comment:
A Promisified version of this should be added to `fs.promises` also

Reply:
The parameter name should be `buffers` to match the comment and the intention.